filament/backend/src/metal/MetalHandles.h

```diff
@@ -322,6 +328,7 @@ class MetalSamplerGroup : public HwSamplerGroup {
     void useResources(id<MTLRenderCommandEncoder> renderPassEncoder);
 
     size_t size;
+    utils::CString debugName;
```

Comment:
because we used a `const char*` u p to this point, and it wasn't safe anyways, there is no point using a CString here. we should document that the name needs to be a string literal.